src/components/LoadingSpinner.js

```diff
@@ -0,0 +1,36 @@
+/*
+		Created this component to display a loading spinner when the user starts
+		to track a currency pair.  It shows for both success cases and failures in case
+		a user is on a slower connection they still know that their data is being fetched.
+		It is reusable and has its information passed in via props.
+ */
```

Comment:
Created this component to display a loading spinner when the user starts to track a currency pair.  It shows for both success cases and failures and in case a user is on a slower connection, they still know that their data is being fetched.  
- It is reusable and has its information passed in via props.